src/components/button/Button.css

```diff
@@ -0,0 +1,42 @@
+.button1 {
+    font-size: 16px;
+    background: #171717;
+    border: 3px solid #171717;
+    color: #FFFFFF;
+    border-radius: 7px;
+    padding: 12px 27px;
+    box-sizing: border-box;
+    width: max-content;
+    min-width: 150px;
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    cursor: pointer;
+}
+
+.button2 {
+    background: #1E47BB;
+    color: #FFFFFF;
+    padding: 15px 12px;
+    border-radius: 7px;
+    font-size: 12px;
+    border: none;
+    cursor: pointer;
+}
+
+.button3 {
```

Comment:
Deze class bevat nu heel veel dubbele CSS in vergelijking met de andere button-classes. Kijk eens of je de CSS efficienter kunt opschrijven met een basisclass die geldt voor alle buttons, en dan extra classes met de properties die anders zijn, zoals font-size, kleur of borders!

Reply:
Hmm ik zie niet heel veel wat exact gelijk blijft, maar ik snap zeker wat je bedoeld.